src/coreclr/tools/aot/ILCompiler.Diagnostics/ILCompiler.Diagnostics.csproj

```diff
@@ -6,7 +6,7 @@
     <IsDotNetFrameworkProductAssembly>true</IsDotNetFrameworkProductAssembly>
     <TargetFramework>$(NetCoreAppToolCurrent)</TargetFramework>
     <DefineConstants>READYTORUN;$(DefineConstants)</DefineConstants>
-    <Platforms>x64;x86;arm;arm64</Platforms>
+    <Platforms>x64;x86;arm;arm64;</Platforms>
```

Comment:
Why is this ';' added here? Did something change in MSBuild that requires lists to be finished with a semicolon?